.php_cs.dist

```diff
@@ -0,0 +1,75 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->in([__DIR__])
+    ->exclude([
+        'vendor',
+    ]);
+
+return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@PhpCsFixer' => true,
+        '@PHP71Migration' => true,
+
+        // required by PSR-12
+        'concat_space' => [
+            'spacing' => 'one',
+        ],
+
+        // disable some too strict rules
+        'phpdoc_types_order' => [
+            'null_adjustment' => 'always_last',
+            'sort_algorithm' => 'none',
+        ],
+        'single_line_throw' => false,
+        'yoda_style' => [
+            'equal' => false,
+            'identical' => false,
+        ],
+        'native_function_invocation' => false,
+        'void_return' => false,
+        'blank_line_before_statement' => [
+            'statements' => ['break', 'continue', 'declare', 'return', 'throw', 'exit'],
+        ],
+        'combine_consecutive_issets' => false,
```

Comment:
if there is a need to disable 30 rules, maybe it's better to explicitly enable full rulesets like PSR2 and maybe few rules on top of it?

Reply:
@keradus I have simple config - https://github.com/mvorisek/TCPDF/tree/psr12_cs_extended - as far I as remember there is one rule that does not finish at all. I belive, in CS fixer, there are several small issues with extreme/exponencial complexity. That is why I opened https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5279

Reply:
@keradus the issue is with `array_indentation`, when not disabled, the CS Fixer never finish (I waited more than 24 hours)

Reply:
https://github.com/tecnickcom/TCPDF/pull/300#discussion_r532117195

Reply:
I do not argue the files are large. But `array_indentation` takes at least 100x more time than the other rules. So I think this is an issue.

Reply:
as i shared in Fixer repo - if you want, do not hesitate to send a PR to make it faster. if you don't want, don't expect someone will do it for you. same time, you got provided an alternative approach - don't fix autogenerated files that you also do not review.